app/models/PropertiesView.scala

```diff
@@ -33,7 +34,8 @@ case class PropertiesView(
       address: Option[String],
       NDRListValuationHistoryItems: Seq[APIValuationHistory],
       parties: Seq[APIParty],
-      agents: Option[Seq[LegacyParty]]) {
+      agents: Option[Seq[LegacyParty]],
```

Comment:
I'm not sure about the naming of `LegacyParty`, but that's for another day

Reply:
Yep, I thought the same when I saw it 😄